src/select.rs

```diff
@@ -33,6 +34,9 @@ pub struct Select<T> {
     cursor: usize,
     initial_value: Option<T>,
     filter: FilteredView<RadioButton<T>>,
+    window_size: usize,
+    window_pos: usize,
+    term_size: Option<Size>,
```

Comment:
@cylewitruk 

Thanks for the patch! :) I believe this part should be used in the `multiselect` as well, thus I would ask to move it into a separate structure, e.g. "`TermSize`", something like I did with `filter.rs` for filtering in the both selection lists.

```rust
TermSize::default()
TermSize::get_size() -> usize
TermSize::set_size(usize)
```

Reply:
@fadeevab agreed, been a bit tight with time the last weeks but I'll get to this eventually :smile:

Reply:
@cylewitruk hello, no rush :) meanwhile, we managed to release few improvements to the multiprogress bar (`insert` and `println`)